examples/mcu-board-support/embassy.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+extern crate alloc;
```

Comment:
this shouldn't be needed as it is already in lib.rs

Reply:
Great - thanks :)